src/code/client.js

```diff
@@ -1170,9 +1183,11 @@ class CloudFileManagerClient {
             const callbackArgs = JSON.stringify(Array.prototype.slice.call(arguments))
             return reply('cfm::event:reply', {eventId: data.eventId, callbackArgs})
         })
-        case 'cfm::event:reply':
-          var event = CLOUDFILEMANAGER_EVENTS[data.eventId]
-          return __guard__(event != null ? event.callback : undefined, x => x.apply(this, JSON.parse(data.callbackArgs)))
+        case 'cfm::event:reply': {
+          const event = CLOUDFILEMANAGER_EVENTS[data.eventId]
+          const callbackData = JSON.parse(data.callbackArgs)
```

Comment:
If `data.callbackArgs` is `undefined` this will throw a parse error - you can move it inline in the next line.